docs/source/usage/index.rst

```diff
@@ -202,7 +220,16 @@ In order to use an external provisioning service, simply attach the node to the
 Additional ESI CLI Actions
 --------------------------
 
-`python-esiclient`_ provides additional commands that combine multiple OpenStack CLI functions into a single action.
+`python-esiclient`_ and `python-esileapclient`_ provide additional commands that combine multiple OpenStack CLI functions into a single action.
```

Comment:
There's no github link for python-esileapclient after I render this doc.

Reply:
Whoops - nice catch. Added!